### ISSUE TITLE
¿Es confusa la metáfora de la pista de audio?

### ISSUE THREAD

**Issue by dcabo** [civio/verba#79]
Extraído de #47: ¿es confuso lo de ordenar las barras como si fueran una pista de audio? La longitud de las barras es proporcional a las cantidades, no estamos distorsionando por ahí. Sí es cierto que las desplazamos, aunque al ser un gráfico en general tan sencillo, el impacto visual puede compensar el esfuerzo adicional para comparar.

Como solución intermedia a quitarlo/ponerlo siempre, se me ocurre que la parte inferior del gráfico, "el reflejo", podría hacerse translúcido al pasar por encima. Así conseguimos el impacto visual inicial, y si alguien explora más, viendo menciones un día/semana, tiene la evolución sin distracciones.

**Comment by llrs** [comment]
Sí, para mí seria mejor un gráfico de barras que no intentar parecer un audio. Quizá el impacto visual se puede conseguir cambiando el estilo o con una imagen en el fondo del gráfico. Por otro lado creo que os hace más fácil anotar picos que están muy seguidos al poder hacerlo tanto por arriba como por debajo y comparar dos búsquedas (aunque he visto que esto no es posible que los genere el usuario #53).
